packages/esbuild-plugin/package.json

```diff
@@ -1,40 +1,71 @@
 {
     "name": "@datadog/esbuild-plugin",
     "packageManager": "yarn@4.0.2",
+    "version": "0.0.13-4",
     "license": "MIT",
     "author": "Datadog",
+    "description": "Datadog ESBuild Plugin.",
+    "keywords": [
+        "datadog",
+        "esbuild",
+        "bundler",
+        "plugin",
+        "unplugin"
+    ],
+    "homepage": "https://github.com/DataDog/build-plugin#readme",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/DataDog/build-plugin",
+        "directory": "packages/esbuild-plugin"
+    },
     "main": "./dist/src/index.js",
+    "module": "./dist/src/index.mjs",
     "exports": {
         ".": "./src/index.ts",
         "./*": "./src/*.ts"
     },
     "publishConfig": {
         "access": "public",
+        "types": "./dist/src/index.d.ts",
         "exports": {
             "./package.json": "./package.json",
-            ".": "./dist/src/index.js",
-            "./*": "./dist/src/*.js"
+            ".": {
+                "import": "./dist/src/index.mjs",
+                "require": "./dist/src/index.js",
+                "types": "./dist/src/index.d.ts"
+            }
         }
     },
     "files": [
         "dist"
     ],
     "scripts": {
-        "build": "yarn clean && tsc",
+        "build": "yarn clean && rollup --config rollup.config.mjs",
         "clean": "rm -rf dist",
         "prepack": "yarn build",
         "typecheck": "tsc --noEmit",
         "watch": "tsc -w"
     },
     "devDependencies": {
-        "typescript": "5.4.3"
-    },
-    "dependencies": {
-        "@datadog/build-plugins-core": "workspace:*",
+        "@babel/core": "7.24.5",
+        "@babel/preset-env": "7.24.5",
+        "@babel/preset-typescript": "7.24.1",
         "@dd/factory": "workspace:*",
-        "esbuild": "0.20.2"
+        "@dd/tools": "workspace:*",
+        "@rollup/plugin-babel": "6.0.4",
+        "@rollup/plugin-commonjs": "25.0.7",
+        "@rollup/plugin-json": "6.1.0",
+        "@rollup/plugin-node-resolve": "15.2.3",
+        "@types/babel__core": "^7",
+        "@types/babel__preset-env": "^7",
+        "esbuild": "0.20.2",
```

Comment:
Note: you could also use `esbuild` as your main builder instead of having to install:
- rollup
- all the rollup plugins
- babel
- all the babel plugins

You can follow this guide for how to setup esbuild: https://esbuild.github.io/getting-started/#bundling-for-node

Reply:
I'm fine with rollup right now, will change if we get to a point where it's necessary to change.